packages/edit-post/src/components/sidebar/plugin-post-publish-panel/index.js

```diff
@@ -73,14 +55,28 @@ const PluginPostPublishPanelFill = ( {
  *
  * @return {WPComponent} The component to be rendered.
  */
+const PluginPostPublishPanel = ( {
+	children,
+	className,
+	title,
+	initialOpen = false,
+	icon,
+} ) => {
+	const context = usePluginContext();
```

Comment:
Optional: since we only use `icon` from the `context`, we might as well destructure it from `context`:

```suggestion
	const { icon: pluginIcon } = usePluginContext();
```